src/spaceone/identity/manager/email_manager.py

```diff
@@ -20,24 +20,28 @@
         "temp_password": "Your password has been changed",
         "verify_email": "Verify your notification email",
         "invite_external_user": "You've been invited to join.",
+        "app_expiration_email": "Your App is expiring",
     },
     "ko": {
         "reset_password": "비밀번호 재설정 안내",
         "temp_password": "임시 비밀번호 발급 안내",
         "verify_email": "알림전용 이메일 계정 인증 안내",
         "invite_external_user": "계정 초대 안내.",
+        "app_expiration_email": "앱 만료 안내",
     },
     "en": {
         "reset_password": "Reset your password",
         "temp_password": "Your password has been changed",
         "verify_email": "Verify your notification email",
         "invite_external_user": "You've been invited to join.",
+        "app_expiration_email": "Your App is expiring",
     },
     "ja": {
         "reset_password": "パスワードリセットのご案内",
         "temp_password": "仮パスワード発行のご案内",
         "verify_email": "通知メールアカウント認証のご案内",
         "invite_external_user": "参加するように招待されました",
+        "app_expiration_email": "",
```

Comment:
In this case, I think temporary use english subject instead of ja.
and copy en template for ja.
```suggestion
        "app_expiration_email": "Your App is expiring",
```